DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsController.cs

```diff
@@ -18,6 +17,10 @@
     [Route("/TrackingSystem/CentreConfiguration/RegistrationPrompts")]
     public class RegistrationPromptsController : Controller
     {
+        public const string DeleteString = "delete";
+        public const string AddPromptString = "addPrompt";
+        public const string NextString = "next";

```

Comment:
Do we need Next & Save? Wouldn't the action always be Save technically?

Reply:
I changed it to distinguish that the button is not save, it is go to next page of the journey